File: python/wejscie_wyjscie.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

def main(args):
    a = input("Podaj liczbę: " )
    print(math.pow(a, 1.0/123)) 
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: cpp/wejscie_wyjscie.cpp
#include <iostream>

using namespace std;

int main(int argc, char **argv)
{
    int a, b;
    a = b = 0;
	cout << "Podaj liczbę:" << endl;
    cin >> a;
    cout << a << endl;
    cout << "Podaj drugą liczbę" << endl;
    cin >> b;
    cout << b;
    
    cout << endl << "Wynik: " << (a*b+a+b)/2;
    
	return 0;
}

